Added question page content and exit page content (for all no answers)

bnaolu opened the pull request:
Added content for issues: "Question page..." and "Exit page content (for all no answers)".

bnaolu committed:
Adding question page content and exit page content (for all no answers)

Added content for issues: "Question page..." and "Exit page content (for all no answers)".

bnaolu commented:
- Added content for question pages
- Added content for exit pages
   - back buttons do not link back to previous pages, yet (still working on this)
Let me know your thoughts on how this is looking.

bnaolu committed:
repositioned stepper in container

pglevy commented on _data/questions-yn.yml:
```diff
@@ -1,36 +1,36 @@
 - number: 1
-  text: Who did this?
+  text: Is your complaint about a covered entity?
```
It might get wordier, but how can we not say "covered entity?"

pglevy commented on _includes/radio.html:
```diff
@@ -11,7 +11,7 @@ <h1>
     <div class="usa-radio">
         <input class="usa-radio__input usa-radio__input--tile" id="{{ item.slug }}{{ forloop.index }}" type="radio" name="{{ item.slug }}" value="{{ answer.path }}">
-        <label class="usa-radio__label" for="{{ item.slug }}{{ forloop.index }}">{{ answer.text }}</label>
+        <label class="usa-radio__label margin-bottom-6" for="{{ item.slug }}{{ forloop.index }}">{{ answer.text }}</label>
```
This utility class doesn't seem to have any effect. What are you trying to accomplish?

bnaolu replied:
LOL absolutely nothing! I was getting lost in all the files! Thank you for catching this, I'll have to be more careful next time.

pglevy commented on _layouts/exit-when.html:
```diff
@@ -0,0 +1,8 @@
+{% include header.html %}
```
Do we need a separate layout for this?

bnaolu replied:
Hmm not sure; what's your opinion on that?

pglevy replied:
I'll look again once it's merged.

pglevy commented on _layouts/exit-who.html:
```diff
@@ -0,0 +1,89 @@
+        <button class="usa-accordion__button"
+          aria-expanded="true"
+          aria-controls="a1">
+          Law Enforcement Agency
```
We should separate the content out from this template. I can help with this if needed. Or you can look at the FAQ page template. I did something similar there.

pglevy commented on _layouts/question-include.html:
```diff
@@ -8,7 +8,7 @@
         This is not a cake nor a cookie
     {% endcase %}
-    <a href="#back" id="backButton" class="usa-button usa-button--outline">Back</a> <a href="#next" id="nextButton" class="usa-button">Next</a>
+    <a href="#back" id="backButton" class="usa-button usa-button--outline margin-top-3">Back</a> <a href="#next" id="nextButton" class="usa-button">Next</a>
```
I did some other stuff with placement of buttons, so we probably don't need this.

pglevy commented on exit-what-1.md:
```diff
@@ -3,6 +3,7 @@ layout: default
 
-Explanation of why not to proceed after the first what question.
+- If you are requesting access to a copy of your health insurance information, your health plan or provider has up to 30 days (sometimes more) to provide the information. Your health insurance plan or provider may require that you make your request in writing.
+- Learn more about your right to access your medical record [here](https://www.hhs.gov/hipaa/for-individuals/medical-records/index.html).
```
Let's make this whole sentence a link (without the period), instead of just "here." In general, avoid linking just the word "here."

pglevy commented on exit-what-1.md:
```diff
@@ -3,6 +3,7 @@ layout: default
 ---
 
-# Exit what 1
+# Things to know:
```
Can we make this heading more descriptive? Maybe something like "Timeline for requesting your health information"?

pglevy commented on exit-what-2.md:
```diff
@@ -3,6 +3,6 @@ layout: default
+# Unfortunately, we can’t complete your request.
 
-Explanation of why not to proceed after the second what question.
+The Department of Health and Human Service’s works only with health information-related violations pertaining to HIPAA. 
```
This is a dead end. Is there anything else we can provide them here in terms of next steps? Or at least clarifying (or restating) why we don't think it applies. Let's try to avoid just referring to "HIPPA" by itself and assume that people know that that means.

pglevy commented on exit-when.md:
```diff
@@ -1,8 +1,10 @@
+# You may still be able to file a complaint.
 
-Explanation of why not to proceed after the when question.
+Please note that under our regulations, we normally only accept complaints that are filed within 180 days of when a complainant knew or should have known that the alleged violation occurred.
```
"complaintant"

pglevy commented on question-1.md:
```diff
@@ -5,7 +5,11 @@ number: 1
+(Doctors, clinics, psychologists, dentists, chiropractors, nursing homes, pharmacies)
+-  Health plans <br>
+(Health insurance companies, HMOs, company health plans, government health programs such as Medicare, Medicaid, and the military and veterans health care programs)
+- Health care clearinghouse <br>
```
I'm still not sure what this is. We may need to provide more information or completely reword this to avoid the word "clearinghouse." Probably makes people think of "sweepstakes," which is probably incorrect and not helpful. :)

pglevy commented on question-3.md:
```diff
@@ -3,4 +3,15 @@ layout: question-include
\ No newline at end of file
+---
+
+**Examples of health information rights::**
```
Extra colon

pglevy commented on question-3.md:
```diff
@@ -3,4 +3,15 @@ layout: question-include
+
+**Examples of health information rights::**
+
+- [HIPAA Privacy Rule ](https://www.hhs.gov/hipaa/for-individuals/guidance-materials-for-consumers/index.html?language=es)<br>
```
This copy works pretty well, but in general, I think it would be more helpful to lead with the explanation. Most people aren't to click or mentally process these official names. But they will understand "Federal law recognizes your rights over your health information and set rules and limits on who can see and receive it." So maybe reversing them and following with the link in parentheses?

pglevy reviewed (changes requested):
Lots of great work here. Nice job! I left a potpourri of comments in individual files. We can sync up if you have questions or need to go into more detail.

bnaolu committed:
Adding changes from PR

bnaolu commented:
Changes made:
- restructured exit-who.html content to better accommodate for accordion feature
   - added content for exit-who.html resources 
- fixed code syntax
   - removed unnecessary button margin class
- fixed sentence wordings throughout
